chore_: market-proxy integration

friofry opened the pull request:
Closes #6740

original pr: #6742

igor-sirotin committed:
test_: use a gorush mock (#6729)

* test_: use a gorush mock

* fix_: linter

osmaczko committed:
fix_: prevent logutils.TestCore flakiness

Output was not deterministic due to timestamp being printed.

friofry committed:
chore_: integrate status market proxy (#6742)

* split ids into chunks
* restore functional tests
* fix hourly charts
* reorder provideres

fixes #6740
# Conflicts:
#	tests-functional/tests/test_wallet_assets.py

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (52)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :x: | d604265e | [#1](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6749/1/) | 2025-07-14 14:51:01 | ~1 min | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6749/1/consoleText) |
| :x: | d604265e | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6749/1/) | 2025-07-14 14:51:21 | ~1 min | `linux` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6749/1/consoleText) |
| :heavy_multiplication_x: | d604265e | [#1](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6749/1/) | 2025-07-14 14:51:27 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6749/1//consoleText) |
| :x: | d604265e | [#1](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6749/1/) | 2025-07-14 14:51:33 | ~1 min | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6749/1/consoleText) |
| :x: | d604265e | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6749/1/) | 2025-07-14 14:51:42 | ~1 min | `macos` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6749/1/consoleText) |
| :x: | d604265e | [#2](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6749/2/) | 2025-07-14 14:51:57 | ~52 sec | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6749/2/consoleText) |
| :heavy_multiplication_x: | d604265e | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6749/2/) | 2025-07-14 14:52:37 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6749/2//consoleText) |
| :x: | d604265e | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6749/2/) | 2025-07-14 14:52:42 | ~1 min | `linux` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6749/2/consoleText) |
| :x: | d604265e | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6749/1/) | 2025-07-14 14:52:49 | ~2 min | `macos` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6749/1/consoleText) |
| :x: | d604265e | [#2](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6749/2/) | 2025-07-14 14:53:02 | ~1 min | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6749/2/consoleText) |
| :x: | d604265e | [#1](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6749/1/) | 2025-07-14 14:53:11 | ~2 min | `windows` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6749/1/consoleText) |
| :x: | d604265e | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6749/2/) | 2025-07-14 14:53:22 | ~1 min | `macos` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6749/2/consoleText) |
| :x: | d604265e | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6749/2/) | 2025-07-14 14:55:26 | ~2 min | `macos` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6749/2/consoleText) |
| :heavy_multiplication_x: | d604265e | [#1](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6749/1/) | 2025-07-14 15:01:03 | ~10 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6749/1//consoleText) |
| :heavy_multiplication_x: | d604265e | [#2](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6749/2/) | 2025-07-14 15:12:32 | ~11 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6749/2//consoleText) |
| | | | | | | |
| :x: | fddba1a6 | [#3](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6749/3/) | 2025-07-14 15:03:20 | ~1 min | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6749/3/consoleText) |
| :x: | fddba1a6 | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6749/3/) | 2025-07-14 15:03:28 | ~1 min | `linux` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6749/3/consoleText) |
| :x: | fddba1a6 | [#3](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6749/3/) | 2025-07-14 15:03:48 | ~2 min | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6749/3/consoleText) |
| :x: | fddba1a6 | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6749/3/) | 2025-07-14 15:03:54 | ~2 min | `macos` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6749/3/consoleText) |
| :heavy_multiplication_x: | fddba1a6 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6749/3/) | 2025-07-14 15:04:09 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6749/3//consoleText) |
| :x: | fddba1a6 | [#2](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6749/2/) | 2025-07-14 15:05:35 | ~3 min | `windows` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6749/2/consoleText) |
| :x: | fddba1a6 | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6749/3/) | 2025-07-14 15:05:44 | ~3 min | `macos` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6749/3/consoleText) |
| :heavy_multiplication_x: | fddba1a6 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6749/3/) | 2025-07-14 15:23:36 | ~10 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6749/3//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 0029dc29 | [#4](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6749/4/) | 2025-07-14 15:18:51 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250714-151653-0029dc-pr6749.aar) |
| :heavy_check_mark: | 0029dc29 | [#4](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6749/4/) | 2025-07-14 15:20:19 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250714-151653-0029dc-pr6749.zip) |
| :heavy_check_mark: | 0029dc29 | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6749/4/) | 2025-07-14 15:20:19 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250714-151656-0029dc-pr6749.zip) |
| :heavy_check_mark: | 0029dc29 | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6749/4/) | 2025-07-14 15:20:54 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250714-151659-0029dc-pr6749.zip) |
| :heavy_check_mark: | 0029dc29 | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6749/4/) | 2025-07-14 15:22:33 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250714-151701-0029dc-pr6749.zip) |
| :heavy_check_mark: | 0029dc29 | [#3](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6749/3/) | 2025-07-14 15:23:03 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250714-151706-0029dc-pr6749.zip) |
| :heavy_check_mark: | 0029dc29 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6749/4/) | 2025-07-14 15:30:43 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6749/4//consoleText) |
| :heavy_multiplication_x: | 0029dc29 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6749/4/) | 2025-07-14 15:47:13 | ~30 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6749/4//consoleText) |
| :heavy_multiplication_x: | 0029dc29 | [#5](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6749/5/) | 2025-07-14 16:50:00 | ~28 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6749/5//consoleText) |
| :heavy_multiplication_x: | 0029dc29 | [#6](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6749/6/) | 2025-07-14 17:25:30 | ~28 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6749/6//consoleText) |
| :heavy_multiplication_x: | 0029dc29 | [#7](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6749/7/) | 2025-07-14 17:59:19 | ~29 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6749/7//consoleText) |
| :heavy_multiplication_x: | 0029dc29 | [#8](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6749/8/) | 2025-07-14 18:37:07 | ~29 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6749/8//consoleText) |
| :heavy_multiplication_x: | 0029dc29 | [#9](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6749/9/) | 2025-07-15 08:25:54 | ~28 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6749/9//consoleText) |
| | | | | | | |
| :heavy_check_mark: | eb16b385 | [#5](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6749/5/) | 2025-07-15 09:40:45 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250715-093650-eb16b3-pr6749.aar) |
| :heavy_check_mark: | eb16b385 | [#5](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6749/5/) | 2025-07-15 09:41:05 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250715-093650-eb16b3-pr6749.zip) |
| :heavy_check_mark: | eb16b385 | [#4](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6749/4/) | 2025-07-15 09:41:32 | ~4 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250715-093704-eb16b3-pr6749.zip) |
| :heavy_check_mark: | eb16b385 | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6749/5/) | 2025-07-15 09:41:44 | ~4 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250715-093653-eb16b3-pr6749.zip) |
| :heavy_check_mark: | eb16b385 | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6749/5/) | 2025-07-15 09:42:52 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250715-093655-eb16b3-pr6749.zip) |
| :heavy_check_mark: | eb16b385 | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6749/5/) | 2025-07-15 09:43:29 | ~6 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250715-093659-eb16b3-pr6749.zip) |
| :heavy_check_mark: | eb16b385 | [#5](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6749/5/) | 2025-07-15 09:46:25 | ~9 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6749/5//consoleText) |
| :heavy_multiplication_x: | eb16b385 | [#10](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6749/10/) | 2025-07-15 10:07:39 | ~30 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6749/10//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 06e0b2dd | [#6](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6749/6/) | 2025-07-15 09:49:24 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250715-094716-06e0b2-pr6749.aar) |
| :heavy_check_mark: | 06e0b2dd | [#6](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6749/6/) | 2025-07-15 09:50:35 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250715-094716-06e0b2-pr6749.zip) |
| :heavy_check_mark: | 06e0b2dd | [#5](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6749/5/) | 2025-07-15 09:51:09 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250715-094729-06e0b2-pr6749.zip) |
| :heavy_check_mark: | 06e0b2dd | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6749/6/) | 2025-07-15 09:51:17 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250715-094719-06e0b2-pr6749.zip) |
| :heavy_check_mark: | 06e0b2dd | [#6](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6749/6/) | 2025-07-15 09:52:29 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250715-094721-06e0b2-pr6749.zip) |
| :heavy_check_mark: | 06e0b2dd | [#6](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6749/6/) | 2025-07-15 09:52:57 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250715-094724-06e0b2-pr6749.zip) |
| :heavy_check_mark: | 06e0b2dd | [#6](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6749/6/) | 2025-07-15 09:54:46 | ~7 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6749/6//consoleText) |
| :heavy_multiplication_x: | 06e0b2dd | [#11](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6749/11/) | 2025-07-15 10:36:29 | ~28 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6749/11//consoleText) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 87c68d1e | [#7](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6749/7/) | 2025-07-15 11:42:55 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250715-114039-87c68d-pr6749.aar) |
| :heavy_check_mark: | 87c68d1e | [#7](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6749/7/) | 2025-07-15 11:44:00 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250715-114042-87c68d-pr6749.zip) |
| :heavy_check_mark: | 87c68d1e | [#7](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6749/7/) | 2025-07-15 11:44:45 | ~4 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250715-114042-87c68d-pr6749.zip) |
| :heavy_check_mark: | 87c68d1e | [#6](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6749/6/) | 2025-07-15 11:45:09 | ~4 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250715-114051-87c68d-pr6749.zip) |
| :heavy_check_mark: | 87c68d1e | [#7](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6749/7/) | 2025-07-15 11:45:51 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250715-114045-87c68d-pr6749.zip) |
| :heavy_check_mark: | 87c68d1e | [#7](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6749/7/) | 2025-07-15 11:46:36 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250715-114047-87c68d-pr6749.zip) |
| :heavy_check_mark: | 87c68d1e | [#7](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6749/7/) | 2025-07-15 11:47:44 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6749/7//consoleText) |
| :heavy_multiplication_x: | 87c68d1e | [#12](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6749/12/) | 2025-07-15 12:11:19 | ~30 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6749/12//consoleText) |
| | | | | | | |
| :heavy_check_mark: | e45f530a | [#8](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6749/8/) | 2025-07-15 14:37:56 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250715-143552-e45f53-pr6749.aar) |
| :heavy_check_mark: | e45f530a | [#8](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6749/8/) | 2025-07-15 14:39:36 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250715-143555-e45f53-pr6749.zip) |
| :heavy_check_mark: | e45f530a | [#8](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6749/8/) | 2025-07-15 14:41:04 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250715-143552-e45f53-pr6749.zip) |
| :heavy_check_mark: | e45f530a | [#8](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6749/8/) | 2025-07-15 14:41:50 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250715-143600-e45f53-pr6749.zip) |
| :heavy_check_mark: | e45f530a | [#7](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6749/7/) | 2025-07-15 14:42:16 | ~6 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250715-143605-e45f53-pr6749.zip) |
| :heavy_check_mark: | e45f530a | [#8](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6749/8/) | 2025-07-15 14:43:05 | ~7 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250715-143558-e45f53-pr6749.zip) |
| :heavy_multiplication_x: | e45f530a | [#8](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6749/8/) | 2025-07-15 14:43:15 | ~7 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6749/8//consoleText) |
| :heavy_check_mark: | e45f530a | [#13](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6749/13/) | 2025-07-15 15:01:50 | ~25 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6749/13//consoleText) |
| :heavy_multiplication_x: | e45f530a | [#9](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6749/9/) | 2025-07-15 15:17:43 | ~7 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6749/9//consoleText) |
| :heavy_check_mark: | e45f530a | [#10](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6749/10/) | 2025-07-15 15:40:12 | ~5 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6749/10//consoleText) |

osmaczko committed:
chore_: ensure NWAKU ports are released

- Exit early if NWAKU ports are already occupied
- Add force flag to NWAKU container removal

osmaczko committed:
chore_: exit tests if any run fails

osmaczko committed:
fix_: use fixed NWAKU version for TestBasicWakuV2

fixes: #4105

jakubgs commented on _assets/ci/Jenkinsfile.tests:
```diff
@@ -85,7 +85,17 @@ pipeline {
     stage('Prep') {
-      steps { /* Go needs to find status-go in GOPATH. */
+      steps {
+        /* Check NWAKU ports before starting containers */
+        sh '''
+          for port in ${NWAKU_REST_PORT} ${NWAKU_TCP_PORT} ${NWAKU_UDP_PORT}; do
+            if lsof -i :$port; then
+              echo "Port $port is already in use! Failing early."
+              exit 1
```
Why do we need this. Ports are adjusted using `EXECUTOR_NUMBER`.
